api/provider/aws/service_read.go

```diff
@@ -124,25 +152,20 @@ func (s *ServiceProvider) populateModelTags(serviceID string, model *models.Serv
 		}
 	}
 
-	deployments := []models.Deployment{}
-	for _, deploy := range model.Deployments {
-		tags, err := s.TagStore.SelectByTypeAndID("deploy", deploy.DeployID)
+	for _, d := range model.Deployments {
+		tags, err := s.TagStore.SelectByTypeAndID("deploy", d.DeployID)
 		if err != nil {
 			return err
 		}
 
-		if tag, ok := tags.WithKey("name").First(); ok {
-			deploy.DeployName = tag.Value
+		if tag, ok := tags.WithKey("deploy_name").First(); ok {
```

Comment:
nothing exists with the key `deploy_name` as far as I'm aware. I think you meant `"name"`?

Reply:
Oop, yeah, my mistake.

✅